pyproject.toml

```diff
@@ -50,6 +50,7 @@ aioshutil = "^1.3"
 beautifulsoup4 = { version = "^4.12.3", optional = true }
 colorama = "^0.4.6"
 cookiecutter = "^2.6.0"
+curl-cffi = { version = "^0.7.1", optional = true }
```

Comment:
Can we also suggest installing the extra on import error?

Reply:
it's there, isn't it? 

https://github.com/apify/crawlee-python/blob/16c67959af41b5043ccf91cf82167e8069f9d63e/src/crawlee/http_clients/__init__.py

or what do u have in mind?

Reply:
Oh right, sorry!